runtime/pack/dist/opt/termdebug/plugin/termdebug.vim

```diff
@@ -687,7 +687,7 @@ function! s:OpenHoverPreview(lines, filetype) abort
             \   'height': height,
             \ })
```

Comment:
try adding `'style': 'minimal', to the `nvim_open_win` "config" parameter. Then the `nvim_win_set_option` calls below aren't needed.

Reply:
Thanks for the suggestion. I made the change, but haven't been able to compile Neovim locally yet (using macOS, need to read up on the documentation), so I don't know if things are working as expected.